docs/guides/faq.html.md

```diff
@@ -0,0 +1,28 @@
+---
+title: FAQ
+subtitle: Plugin creation
+layout: guide
+order: 0
+---
+
+This ensures that Danger only runs when you have the environment variables set up to run.
+
+### Can Danger comment inline on an PR?
```

Comment:
[This GitHub documentation](https://help.github.com/articles/commenting-on-a-pull-request/#adding-line-comments-to-a-pull-request) expresses what an inline comment means in this section. Could this be stated more clearly?